core/server/src/main/java/alluxio/worker/block/BlockWorkerClientRestServiceHandler.java

```diff
@@ -68,6 +68,9 @@
   private final StorageTierAssoc mStorageTierAssoc =
       new WorkerStorageTierAssoc(WorkerContext.getConf());
 
+  // prevent instantiation
```

Comment:
this needs to have a public constructor because it is instantiated by jersey framework when creating REST endpoint handlers